scripts/install.sh

```diff
@@ -0,0 +1,11 @@
+os=`echo "$(uname -s)_$(uname -m)"`
+if [ $os == "Linux_x86_64" ] || [ $os == "Linux_armv6" ] || [ $os == "Windows_x86_64" ] || [ $os == "Darwin_x86_64" ] || [ $os == "Linux_arm64" ] || [ $os == "Windows_i386" ] || [ $os == "Linux_i386" ]]
```

Comment:
I don't think this will work with Windows as the script uses UNIX commands only.

Reply:
Yes, it's not working in windows. I think it's better creating an executable file from python using pyinstaller.

Reply:
Source code designed for Unix-like operating systems may be compiled and run natively with minimal modification on windows using [Cygwin](https://www.cygwin.com/). It will be better I feel instead of creating another source code for windows.

Reply:
I don't want to provide any script for Windows installation as it would need further maintenance.